specification/iothub/resource-manager/Microsoft.Devices/stable/2019-03-22/iothub.json

```diff
@@ -0,0 +1,3482 @@
+          "type": "string",
+          "readOnly": true
+        },
+        "expiry": {
+          "description": "The certificate's expiration date and time.",
+          "type": "string",
+          "format": "date-time-rfc1123",
+          "readOnly": true
+        },
+        "thumbprint": {
+          "description": "The certificate's thumbprint.",
+          "type": "string",
+          "readOnly": true
+        },
+        "isVerified": {
+          "description": "Determines whether certificate has been verified.",
+          "type": "boolean",
+          "readOnly": true
+        },
+        "created": {
+          "description": "The certificate's create date and time.",
+          "type": "string",
+          "format": "date-time-rfc1123",
+          "readOnly": true
+        },
+        "updated": {
+          "description": "The certificate's last update date and time.",
+          "type": "string",
+          "format": "date-time-rfc1123",
+          "readOnly": true
+        },
+        "certificate": {
+          "description": "The certificate content",
+          "type": "string"
+        }
+      }
+    },
+    "CertificatePropertiesWithNonce": {
+      "description": "The description of an X509 CA Certificate including the challenge nonce issued for the Proof-Of-Possession flow.",
+      "type": "object",
+      "properties": {
+        "subject": {
+          "description": "The certificate's subject name.",
+          "type": "string",
+          "readOnly": true
+        },
+        "expiry": {
+          "description": "The certificate's expiration date and time.",
+          "type": "string",
+          "format": "date-time-rfc1123",
+          "readOnly": true
+        },
+        "thumbprint": {
+          "description": "The certificate's thumbprint.",
+          "type": "string",
+          "readOnly": true
+        },
+        "isVerified": {
+          "description": "Determines whether certificate has been verified.",
+          "type": "boolean",
+          "readOnly": true
+        },
+        "created": {
+          "description": "The certificate's create date and time.",
+          "type": "string",
+          "format": "date-time-rfc1123",
+          "readOnly": true
+        },
+        "updated": {
+          "description": "The certificate's last update date and time.",
+          "type": "string",
+          "format": "date-time-rfc1123",
+          "readOnly": true
+        },
+        "verificationCode": {
+          "description": "The certificate's verification code that will be used for proof of possession.",
+          "type": "string",
+          "readOnly": true
+        },
+        "certificate": {
+          "description": "The certificate content",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "IotHubProperties": {
+      "description": "The properties of an IoT hub.",
+      "type": "object",
+      "pr
```

Comment:
```suggestion
         "description": "The name of the Azure region",
```

Reply:
Wait, looks like this feature is buggy

Reply:
@reshantchandra don't use the commit suggestion feature, I think it's going to modify the wrong line

Reply:
Please update the json directly

Reply:
The patch is probably right. GitHub just messes up these line comments when it comes to large files. It usually looks OK on the files tab